operators/aqua/2022.4.0/manifests/aqua-operator.v2022.4.0.clusterserviceversion.yaml

```diff
@@ -4,14 +4,12 @@ metadata:
   name: aqua-operator.v2022.4.0
   namespace: placeholder
   annotations:
-    # Setting olm.maxOpenShiftVersion automatically
-    # This property was added via an automatic process since it was possible to identify that this distribution uses API(s),
-    # which will be removed in the k8s version 1.22 and OpenShift version OCP 4.9. Then, it will prevent OCP users to
-    # upgrade their cluster to 4.9 before they have installed in their current clusters a version of your operator that
-    # is compatible with it. Please, ensure that your project is no longer using these API(s) and that you start to
-    # distribute solutions which is compatible with Openshift 4.9.
-    # For further information, check the README of this repository.
-    olm.properties: '[{"type": "olm.maxOpenShiftVersion", "value": "4.9"}]'
+    # Setting operatorhub.io/ui-metadata-max-k8s-version annotation automatically
+    # The following is an informative annotation to let its users know that this distribution
+    # uses removed APIs in 1.22 and will not work on K8S clusters 1.22+.
+    # Following the findings for this distribution:
+    # this bundle is using APIs which were deprecated and removed in v1.22. More info: https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-22. Migrate the API(s) for CRD: (["aquacsps.operator.aquasec.com" "aquadatabases.operator.aquasec.com" "aquaenforcers.operator.aquasec.com" "aquagateways.operator.aquasec.com" "aquakubeenforcers.operator.aquasec.com" "aquascanners.operator.aquasec.com" "aquaservers.operator.aquasec.com"])
+    operatorhub.io/ui-metadata-max-k8s-version: "1.22"
```

Comment:
@yossig-aquasec:
It was not added automatically.
We never added 1.22 because it had no reason for that. 

Could you please fix this one as well.